lib/ama_layout/navigation.rb

```diff
@@ -33,6 +34,18 @@ def navigation_items
       YAML.load ERB.new(File.read nav_file_path).result
     end
 
+    def display_name
+      @display_name && @display_name.titleize || email
```

Comment:
Personally I'd rather see a ternary than the logical equivalent.

`display_email? ? email : @display_name.titleize`